apps/frontend-nuxt/pages/blog/index.vue

```diff
@@ -0,0 +1,36 @@
+<script setup lang="ts">
+import { BlogRsc } from "@effect-ts-app/boilerplate-client"
+
+const blogClient = clientFor(BlogRsc)
```

Comment:
How's a client different from a layer/effect/service?

Reply:
It's an API client, so an object with either
- methods to call with Request Input, returning Effects to run which will return Response Output
or
- Effects to run which will return Response Output